### DISCUSSION TITLE
Does one reloadable layer need to be reloaded after previous one reloaded

### DISCUSSION THREAD

**Discussion by tiany7** [tokio-rs/tracing#2693] [Q&A] [answered]
Hi everyone! 
I now have two layers of reloadable layers in my tracing log, I would like to know if one of my layers is reloaded, do I need to reload the adjacent layer as well.
Any comment is greatly appreciated!!
Thanks

**Comment by hawkw** [comment, marked as answer]
You only need to reload the layer you are actively changing. If you aren't making changes to another layer, it doesn't need to be reloaded.

**Reply by tiany7** [reply]
Thank you so much!
